compare.go

```diff
@@ -72,6 +72,63 @@ func platformVector(platform specs.Platform) []specs.Platform {
 		if variant == "" {
 			variant = "v8"
 		}
+
+		majorVariant, minorVariant, hasMinor := strings.Cut(variant, ".")
+		if armMajor, err := strconv.Atoi(strings.TrimPrefix(majorVariant, "v")); err == nil && armMajor >= 8 {
+			armMinor := 0
+			if len(variant) == 4 {
+				if minor, err := strconv.Atoi(minorVariant); err == nil && hasMinor {
+					armMinor = minor
+				}
+			}
+
+			if armMajor == 9 {
+				for minor := armMinor - 1; minor >= 0; minor-- {
+					arm64Variant := "v" + strconv.Itoa(armMajor) + "." + strconv.Itoa(minor)
+					if minor == 0 {
+						arm64Variant = "v" + strconv.Itoa(armMajor)
+					}
+					vector = append(vector, specs.Platform{
+						Architecture: platform.Architecture,
+						OS:           platform.OS,
+						OSVersion:    platform.OSVersion,
+						OSFeatures:   platform.OSFeatures,
+						Variant:      arm64Variant,
+					})
+				}
+
+				// v9.0 diverged from v8.5, meaning that v9.x is compatible with v8.{x+5}
+				armMinor = armMinor + 5
```

Comment:
Should this stop at 8.9?

Reply:
Fixed in fd2a630aedb409576b25be2f23730ddc3e099eb1